tests/__fixtures__/bags/bag.fixtures.ts

```diff
@@ -47,6 +47,23 @@ export const mockBagItemData = {
   quantity: 99,
 };
 
+export const mockBagPromocodesResponse = {
+  promoCodesInformation: [
+    {
+      promoCode: 'code-1',
+      isValid: true,
+      error: {
```

Comment:
Is the error set in the response when `isValid` is true? If not, then it is better to have the mock be as realistic as possible with what the endpoint returns. If you need to test a promocode that contains an error, add another element to the array with `isValid` equal to `false` and the error set.

Reply:
Yes, it makes sense. I removed the error object for a valid mock